app/controllers/vm_common.rb

```diff
@@ -708,8 +708,16 @@ def policies
     build_policy_tree(@polArr)
     @edit = session[:edit] if session[:edit]
     if @edit && @edit[:explorer]
-      @in_a_form = true
-      replace_right_cell
+      if session[:policies].empty?
+        add_flash(_("No policies were selected for Policy Simulation."), :error)
```

Comment:
@ZitaNemeckova  can we you use `render_flash` as we have here https://github.com/ManageIQ/manageiq/blob/master/app/controllers/application_controller.rb#L424 instead of lines 712 - 716.